FixMatrix/FixMatrix.cpp

```diff
@@ -0,0 +1,208 @@
+#include <cstdlib>
+#include <iostream>
+#include <stdexcept>
+
+namespace {
+
+    class matrix {
+        int Row, Col;
+        double **Value;
+    public:
+
+        matrix(unsigned int, unsigned int);
+
+        matrix(const matrix &);
+
+        int GetRow();
+
+        int GetCol();
+
+
+        friend bool operator==(const matrix &m1, const matrix &m2);
+
+        friend bool operator!=(const matrix &m1, const matrix &m2);
```

Comment:
name the operands as "lhs" and "rhs". Well known shorthand:
https://en.wikipedia.org/wiki/Sides_of_an_equation